.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Interesting. We only call babel in one place in package.json, so this is easy to remove.